### PR TITLE
Test folders

### PR DESCRIPTION
**Pull request by UziTech**
**Marked version:** 0.3.19

## Description

- Put cm tests in spec folder
image

- Run all tests to make sure failing tests fail

## Contributor

- [x] Test(s) exist to ensure functionality and minimize regression (if no tests added, list tests covering this PR).
- [ ] If submitting new feature, it has been documented in the appropriate places.

## Committer

In most cases, this should be a different person than the contributor.

- [ ] Draft GitHub release notes have been updated.
- [ ] CI is green (no forced merge required).
- [ ] Merge PR

### CODE REVIEW

File: test/specs/commonmark/commonmark-spec.js [hunk 1]
```diff
@@ -49,6 +50,12 @@ var messenger = new Messenger();
 |Soft line breaks            |1 of 2   |50%       |
 */
 
+beforeEach(function () {
+  marked.setOptions({
+    sanitize: false
+  });
+});
+
```
Comment (UziTech) [inline test/specs/commonmark/commonmark-spec.js]: We can remove this after #1212 is merged

File: test/specs/commonmark/commonmark-spec.js [hunk 2]
```diff
@@ -169,7 +176,7 @@ describe('CommonMark 0.28 HTML blocks', function() {
   var section = 'HTML blocks';
 
   // var shouldPassButFails = [];
-  var shouldPassButFails = [132, 126, 147, 121, 124, 120, 153, 133, 119, 127, 117, 118, 141, 116, 158, 122, 123, 128, 143, 130, 136, 137, 140, 125, 134, 131, 144, 145, 146, 148, 139, 149, 129, 156, 135, 138, 142, 150, 151, 152, 154, 155, 157];
+  var shouldPassButFails = [132, 126, 147, 124, 120, 153, 133, 119, 127, 118, 141, 116, 158, 123, 143, 130, 137, 140, 125, 134, 131, 144, 145, 148, 139, 149, 129, 156, 135, 138, 155];
```
Comment (joshbruce) [inline test/specs/commonmark/commonmark-spec.js]: What change to the lib has been made to make them pass? Honestly curious.

Comment (UziTech) [reply test/specs/commonmark/commonmark-spec.js]: I think just resetting the defaults

**Comment by UziTech** [comment]
Who wants to do the honors?

**Comment by joshbruce** [comment]
I will. Looks like we're waiting on Travis. Then we should probably head back over to #1135 to see what happened if anything.

**Pull request merged**